### PR TITLE
feat: add newsletter component

### LINKED ISSUE #9: Create Newsletter Signup section/form

### PR DESCRIPTION
**Pull request by arian81**
resolves #9

### CODE REVIEW

File: src/components/Newsletter.tsx [hunk 1]
```diff
@@ -0,0 +1,182 @@
+const Newsletter = () => {
+  return (
+    <div className="relative mx-auto mt-10 max-w-2xl">
+      <div className="letter absolute left-1/2 right-0 z-0 w-3/4 -translate-x-1/2 transform cursor-pointer md:bottom-48">
+        <svg
```
Comment (Krish120003) [inline src/components/Newsletter.tsx]: The SVG should be imported or using Next/Image like the other sections

File: src/components/Newsletter.tsx [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+                    fill="#533688"
+                  />
+                </svg>
+
+                <svg
```
Comment (Krish120003) [inline src/components/Newsletter.tsx]: Same with this SVG, please import or using Next/Image like the other sections

File: src/sections/Footer.tsx [hunk 3]
```diff
@@ -1,10 +1,14 @@
 import React from "react";
 import Image from "next/image";
+import Newsletter from "~/components/Newsletter";
 
 const Footer = () => {
   return (
     <footer className="grid min-h-screen grid-cols-1 grid-rows-1 bg-red-500">
       <div className="grid-col relative col-start-1 row-start-1 overflow-clip object-cover p-16">
+        {/* <div className="z-[-1000] mt-32">
```
Comment (Krish120003) [inline src/sections/Footer.tsx]: Why is it commented out?

File: src/sections/Footer.tsx [hunk 4]
```diff
@@ -13,9 +17,9 @@ const Footer = () => {
           height={1080}
         />
       </div>
-      <div className="z-10 col-start-1 row-start-1">
+      {/* <div className="z-10 col-start-1 row-start-1">
```
Comment (Krish120003) [inline src/sections/Footer.tsx]: Unnecessary comment

File: src/sections/Footer.tsx [hunk 5]
```diff
@@ -1,10 +1,14 @@
 import React from "react";
 import Image from "next/image";
+import Newsletter from "~/components/Newsletter";
```
Comment (Krish120003) [inline src/sections/Footer.tsx]: Newsletter signup should be a section, not a component

**Review by Krish120003 — CHANGES_REQUESTED** [review]

File: src/sections/Footer.tsx [hunk 6]
```diff
@@ -14,6 +14,109 @@ const Footer = () => {
     <footer>
       <div className="h-28 bg-gradient-to-b from-transparent to-[#efdeff]" />
       <div className="mx-auto flex flex-col items-center bg-[#efdeff]">
+        <div className="relative mx-auto mt-10 max-w-2xl">
```
Comment (Krish120003) [inline src/sections/Footer.tsx]: The newsletter signup should be it's own section, not in the footer 😅

**Review by Krish120003 — DISMISSED** [review]

**Comment by Krish120003** [comment]
@arian81 There's a major discrepancy between sizes of this form on the Figma vs the actual site. Please confirm with @Shivermist how big the form should be and either update the site or have the design files updated.

**Comment by Krish120003** [comment]
image
There's also a weird breakpoint where the rounding is removed completely, but the page isnt small enough for the form dialog to be seamlessly flush with the edges of the page. Could you update the styling to handle this?

**Comment by arian81** [comment]
> image There's also a weird breakpoint where the rounding is removed completely, but the page isnt small enough for the form dialog to be seamlessly flush with the edges of the page. Could you update the styling to handle this?

not sure what to do about this because it's not caught by the tailwind breakpoints. it doesn't happen in any of the device sizes that I've checked so I think it should be good as is ?

**Review by kabirsg — APPROVED** [review]
LGTM

**Pull request merged**